docs/semgrep-assistant/analyze.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: analyze
+title: Analyze findings
+hide_title: true
+description: Learn how to analyze findings with Assistant and view its results.
+tags:
+  - Deployment
+  - Semgrep Assistant
+---
+
+# Analyze findings with Semgrep Assistant
+
+Once you've [enabled Assistant](/docs/semgrep-assistant/getting-started), you can use the **Analyze** button on the [Findings page](/semgrep-code/findings) to trigger all Assistant functions, including autofix, auto-triage, and component tagging, on existing findings.
+
+![Assistant Analyze button on Findings page](/img/scp-assistant.png#md-width)
+
+To analyze your findings with Assistant:
+
+1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, select the findings that you want Assistant to analyze.
+2. Click **Analyze**.
+3. In the confirmation window that appears, confirm that you want to analyze your findings with Assistant.
+
+After Assistant performs these functions, you can see your results on the **Findings** page using the **Recommendation** or **Component** filters. When viewing your findings, you can see false positive and true positive recommendations when viewing the finding details pages if you choose **No Grouping** instead of **Group by Rule**.
+
+The amount of time required to analyze your findings varies, but the UI displays a notification that provides an estimated wait time.
+
+:::info
+- For Team tier users with less than 10 contributors: There is a cap of 50 Assistant runs per month using the **Analyze** button.
+- For Team or Enterprise users with an active subscription: There is a cap of 10,000 Assistant runs per month using the **Analyze** button. It is rate-limited to 1,000 Assistant runs per hour.
+- For users of any tier: Assistant runs against pull requests and merge requests do not count against this limit.
+:::
+
+## View recommendations
+
+You can [view all of Semgrep Assistant's recommendations](/semgrep-code/findings/#filter-findings) by going to the Semgrep **Findings** page and filtering by **Recommendation** or **Component**.
+
+## Feedback
+
+Semgrep Assistant prompts you for feedback whenever it suggests that a finding is a false positive. Because Assistant content is generated by language models (LLMs), your feedback helps the Semgrep team improve Assistant.
+
```

Comment:
The following set of bullets starting from line 41 to 43 need a sentence to introduce them or explain their unifying theme.